DHI inline patch (druid 36.0): bump vertx-core for CVE-2026-6860 and CVE-2026-1002

Advisories require vertx-core 4.5.27+ (CVE-2026-6860) and 4.5.24+ (CVE-2026-1002):
https://scout.docker.com/v/CVE-2026-6860
https://scout.docker.com/v/CVE-2026-1002

diff --git a/extensions-core/kubernetes-overlord-extensions/pom.xml b/extensions-core/kubernetes-overlord-extensions/pom.xml
index 1111111..2222222 100644
--- a/extensions-core/kubernetes-overlord-extensions/pom.xml
+++ b/extensions-core/kubernetes-overlord-extensions/pom.xml
@@ -95,7 +95,7 @@
     <dependency>
       <groupId>io.vertx</groupId>
       <artifactId>vertx-core</artifactId>
-      <version>4.5.14</version>
+      <version>4.5.28</version>
     </dependency>
     <dependency>
       <groupId>javax.ws.rs</groupId>
